src/main/kotlin/no/nav/syfo/db/VaultCredentialService.kt

```diff
@@ -57,6 +57,7 @@ data class RenewCredentialsTaskData(
     val role: Role
 )
 
+// TODO: Er dette egentlig det samme som VaultSecrets i Environment.kt?
```

Comment:
Nei, jeg tror det er to forskjellige ting :) Dette her er vel for oppsettet som brukes for å hente databasecredentials fra vault? Den i Environment er de vanlige secretene vi henter fra vault :)

Reply:
Okay. Grunnen til at jeg spør er at i syfosmmottak ligger en klasse som heter VaultCredentials i Environment.tk som ser ut til å være basically det samme som vår VaultSecrets, så jeg stusset litt. https://github.com/navikt/syfosmmottak/blob/master/src/main/kotlin/no/nav/syfo/Environment.kt

Antakeligvis har det ikke så mye i så, det er jo bare data-klasser :)

Reply:
Den du lenker til er det samme som det som heter VaultSecrets i egenmeldt-sykmelding-backend :) Vi bruker vault til to ting i egenmeldt-backenden: Hente secrets (det er det som ligger i Environment), og å hente og fornye tilgang til databasen (det er den som ligger i VaultCredentialService).